### PR TITLE
Feature/ui components

### PR DESCRIPTION
**Pull request by JessBrownless**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
   Bare Bones
 </h1>
 
-<p>A bare bones website for client development</p>
+<p>A bare bones website for client development lorem ipsum dolor sit amet.</p>
```
Comment (sonnybooth-hexsu) [inline README.md]: Remove

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "bare-bones",
   "private": true,
-  "description": "A bare bones website for client development",
+  "description": "A bare bones website for client development lorem ipsum dolor sit amet.",
```
Comment (sonnybooth-hexsu) [inline package.json]: Remove

File: src/components/Footer/footer.module.css [hunk 3]
```diff
@@ -3,62 +3,3 @@
 @value colorPrimary, colorBlack, colorWhite from colors;
```
Comment (sonnybooth-hexsu) [inline src/components/Footer/footer.module.css]: Might as well delete the file then

File: src/components/Footer/footer.tsx [hunk 4]
```diff
@@ -1,6 +1,8 @@
 import React from "react"
 import styles from "./footer.module.css"
```
Comment (sonnybooth-hexsu) [inline src/components/Footer/footer.tsx:2]: Why space?

File: src/components/Footer/footer.tsx [hunk 5]
```diff
@@ -18,29 +20,102 @@ export const Footer = ({
...
-                    <li data-testid="footerNavigationLink" key={i}>
-                      <a href={page}>{name}</a>
-                    </li>
-                  ))}
-                </ol>
-              </nav>
+      <div className="bg-black">
+        <div className="container mx-auto pl-6 pr-6 pt-16 pb-16">
+          <div className="pl-4 pr-4">
+            <div className="grid grid-cols-1 gap-10 grid-col md:gap-6 md:grid-cols-3 lg:grid-cols-12">
```
Comment (sonnybooth-hexsu) [inline src/components/Footer/footer.tsx]: Do we need grid-col?

File: src/components/Footer/footer.tsx [hunk 6]
```diff
@@ -18,29 +20,102 @@ export const Footer = ({
...
-            <div className={styles.footerMeta}>
-              <p>{email}</p>
-              <p>{telephone}</p>
+          </div>
+        </div>
+        <div className="bg-gray-700">
+          <div className="container mx-auto pl-6 pr-6">
+            <div className="pl-4 pr-4">
+              <p className="text-xs pt-2 pb-2 text-center text-white lg:text-left">
+                ©2018 - Bare Bones | All right reserved.
```
Comment (sonnybooth-hexsu) [inline src/components/Footer/footer.tsx:116]: 2018? weirdo

**Review by sonnybooth-hexsu — CHANGES_REQUESTED** [review]
cmon

File: src/components/Footer/footer.tsx (hunk 6 above)
Comment (JessBrownless) [reply src/components/Footer/footer.tsx:116]: Thats a special piece of code for you to do

**Review by sonnybooth-hexsu — APPROVED** [review]

**Pull request merged**